Action failing: Command failed: git describe --tags --abbrev=0

The same commits in my repo which were previously passing are now failing

```
Run goreleaser/goreleaser-action@v1
✅ GoReleaser version found: v0.126.0
⬇️ Downloading https://github.com/goreleaser/goreleaser/releases/download/v0.126.0/goreleaser_Linux_x86_64.tar.gz...
📦 Extracting GoReleaser...
/bin/tar --version
tar (GNU tar) 1.29
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
/bin/tar xz --warning=no-unknown-keyword -C /home/runner/work/_temp/04821171-ce64-41cd-84e3-68cb9f85226e -f /tmp/goreleaser-vE1CQY/goreleaser_Linux_x86_64.tar.gz
fatal: No names found, cannot describe anything.
##[error]Command failed: git describe --tags --abbrev=0
fatal: No names found, cannot describe anything.

##[error]Node run failed with exit code 1
```

--- a/lib/git.js
+++ b/lib/git.js
@@ -37,7 +37,12 @@ function isTagDirty(currentTag) {
 exports.isTagDirty = isTagDirty;
 function getTag() {
     return __awaiter(this, void 0, void 0, function* () {
-        return yield git(['describe', '--tags', '--abbrev=0']);
+        try {
+            return yield git(['describe', '--tags', '--abbrev=0']);
+        }
+        catch (err) {
+            return '';
+        }
     });
 }
 exports.getTag = getTag;

--- a/src/git.ts
+++ b/src/git.ts
@@ -17,7 +17,11 @@ export async function isTagDirty(currentTag: string): Promise<boolean> {
 }
 
 export async function getTag(): Promise<string> {
-  return await git(['describe', '--tags', '--abbrev=0']);
+  try {
+    return await git(['describe', '--tags', '--abbrev=0']);
+  } catch (err) {
+    return '';
+  }
 }
 
 export async function getShortCommit(): Promise<string> {

